README.md

```diff
@@ -40,364 +43,346 @@ some kind of discriminant:
-- `Enum.create(members)` - Creates an _enum object_ using the property keys and values of `members`
-  as the _enum members_ for the new enum.
-- `Enum.flags(descriptor)` - A built-in decorator that modifies the _enum object_ in the following ways:
-  - The auto-increment behavior is changed to shift the current auto-increment value left by 1. 
-  - The `@@parseEnum` method is modified to parse a comma-separated string and OR the resulting values
-    together. If no corresponding name can be found and the name can be successfully coerced to a number,
-    that number is OR'ed with the result.
-  - The `@@formatEnum` method is modified to convert a bitwise combination of flag values into a comma
-    separated string of corresponding names. If no corresponding name can be found, the SV of the 
-    bits is appended to the string.
+TypeScript's `enum` supports auto-initialization of enum members:
 
 ```ts
-let Enum: {
-  keys(E: object): IterableIterator<string | symbol>;
-  values(E: object): IterableIterator<any>;
-  entries(E: object): IterableIterator<[string | symbol, any]>;
-  has(E: object, key: string | symbol): boolean;
-  hasValue(E: object, value: any): boolean;
-  getName(E: object, value: any): string | undefined;
-  format(E: object, value: any): string | symbol | undefined;
-  parse(E: object, value: string): any;
-  create(members: object): object;
-  flags(descriptor: EnumDescriptor): EnumDescriptor;
-};
+enum Numbers {
+  zero, // 0
+  one,  // 1
+  two   // 2
+}
 ```
 
-<!--
-# Grammar
+However, this behavior is contentious amongst some TC39 delegates and has been removed from this proposal. TypeScript
+will continue to support auto-initializiation due to its frequent use within the developer community, but would emit
+explicit initializers to JavaScript. It is possible that another form of auto-initializiation may be introduced in the
+future that could be utilized by both TypeScript and ECMAScript.
 
-```grammarkdown
-```
--->
+## Declaration Merging
 
-# Examples
+TypeScript (as of v5.8) allows `enum` declarations to merge with other `enum` (and `namespace`) declarations with the
+same name. This is not a desirable feature for ECMAScript enums and will not be supported. TypeScript is considering
+[deprecating this feature](https://github.com/microsoft/TypeScript/issues/54500#issuecomment-2770170732) in
+TypeScript 6.0.
 
-<!-- Examples of the proposal -->
+## Reverse Mapping
+
+TypeScript currently supports reverse-mapping enum values back to enum member names using `E[value]`, but only for
+`Number` values. This limitation is intended to avoid collisions for `String`-valued enum members that could potentially
+overwrite other members. While this information is invaluable for debugging, diagnostics, formatting, and serialization,
+it is far less frequently used compared to `enum` on the whole.
+
+To avoid this inconsistency, we propose instead propose introducing reverse mapping by way of the `@@iterator` built-in
```

Comment:
```suggestion
To avoid this inconsistency, we propose instead propose introducing general iteration by way of the `@@iterator` built-in
```

From a user's perspective there's nothing "reversed" in the proposed JS feature.  So I don't think we should sell it that way.  The "reverse" framing only refers to the older TS feature that the new iteration feature supersedes.